codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -1249,6 +1154,27 @@ void removeFromActiveStreams(DefaultStream stream, Iterator<?> itr) {
         boolean allowModifications() {
             return pendingIterations == 0;
```

Comment:
Does `pendingIterations` need to be marked as `volatile` ? In case of modified by several threads concurrently?